Packet++/src/Asn1Codec.cpp

```diff
@@ -404,7 +404,7 @@ namespace pcpp
 	{
 		if (m_EncodedValue != nullptr)
 		{
-			decodeValue(m_EncodedValue, true);
```

Comment:
the `true` here doesn't make any effect, since all `decodeValue`s don't use `lazy` argument.